packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -69,20 +70,51 @@ export class ArtifactCreator {
 
       mkdirp.sync(scenarioOutputDirectory);
 
-      const screenshot = await this.captureScreenshot(
-          'model-viewer',
-          scenarioName,
-          dimensions,
-          join(scenarioOutputDirectory, 'model-viewer.png'));
+      let screenshot;
+      try {
+        screenshot = await this.captureScreenshot(
+            'model-viewer',
+            scenarioName,
+            dimensions,
+            join(scenarioOutputDirectory, 'model-viewer.png'));
+      } catch (error) {
+        throw new Error(`❌ Failed to capture model-viewer's screenshot of ${
+            scenarioName}. Error message: ${error.message}`);
+      }
 
       if (screenshot == null) {
-        console.log(`🚨 Failed to capture screenshot`);
-        continue;
+        throw new Error(`❌ Model viewer's screenshot of ${
+            scenarioName} are not capture correctly! (the value is null)`);
       }
 
       const analysisResults =
           await this.analyze(screenshot, goldens, scenario, dimensions);
 
+      const modelViewerIndex = 0;
+      const modelViewerRmsInDb =
+          toDecibel(analysisResults[modelViewerIndex].rmsDistanceRatio);
+      // add additional check on whether all renders give bad metrics. Tha
+      // probably means the model-viewer screenshot captured in the fidelity
+      // test maybe completely white or transparent
```

Comment:
This is too approximate. If we're going to have this check, it needs to be part of the analysis where it checks that every pixels is (1,1,1) or (0,0,0).

Reply:
If that's looking complicated, we can save it for a future PR.

Reply:
I think you addressed my feedback here, but the comment looks out of date?